### PR TITLE
device: Added support for EdgeCore ES3510MA

### PR DESCRIPTION
**Pull request by backslash7**
Added support for EdgeCore ES3510MA switch. Tested on fw version 1.5.2.14.

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`

Fixes: #9128

### CODE REVIEW

**Comment by laf** [comment]
Can you add test units as well please: https://docs.librenms.org/#Developing/os/Test-Units/#example-workflow

File: includes/definitions/edgecos.yaml [hunk 1]
```diff
@@ -10,6 +10,7 @@ over:
 discovery:
     - sysObjectID:
         - .1.3.6.1.4.1.259.6.10.94
+        - .1.3.6.1.4.1.259.8.1.11.105
```
Comment (murrant) [inline includes/definitions/edgecos.yaml]: Can you shorten this to `.1.3.6.1.4.1.259.8.1.11` ?

https://github.com/librenms/librenms/issues/9128
